src/supervisor.tex

```diff
@@ -1741,7 +1932,7 @@ \subsection{Addressing and Memory Protection}
 \instbit{1} &
 \instbit{0} \\
 \hline
-\multicolumn{1}{|c|}{\it Reserved} &
+\multicolumn{1}{|c|}{\it attributes} &
```

Comment:
Possibly orphaned, there is no other mention of attributes in this draft?

Reply:
The virtual memory TG is considering adding other attributes, but since we haven't finalized anything, I've changed this back to Reserved